### ISSUE TITLE
Problem with downloading files 

### ISSUE THREAD

**Issue by omer187-ir** [elishacloud/Silent-Hill-2-Enhancements#1068]
Hi, I can't download files from Wizard after minutes, getting time out tried another internet, hotspot but randomly times out

**Comment by Polymega** [comment]
Has this been resolved for you?

**Comment by omer187-ir** [comment]
yes after many tries about switching networks and using VPN it fixed

**Comment by Polymega** [comment]
We're still trying to resolve this issue. Can you please try this test file for me? Here's what you do:

1. Go to your `SILENT HILL 2` game folder and make a _backup_ of your `SH2EEsetup.exe` file in there.
2. Once backed up to another location, download this new version of `SH2EEsetup.exe` here: [SH2EEsetup_Download_Fix_Test.zip](https://github.com/user-attachments/files/17652785/SH2EEsetup_Download_Fix_Test.zip)
3. Unzip this package and place the new `SH2EEsetup.exe` within your `SILENT HILL 2` game folder (where the previous `SH2EEsetup.exe` existed).
4. Run the new `SH2EEsetup.exe` and let us know if you can now successfully download the package files or not.

Because this is a new test file, you might get a [false positive](https://allaboutcookies.org/antivirus-false-positive) warning about it from your antivirus vendor. The file is safe but, if you're still unsure, you can submit the file for review to Windows Defender, BitDefender, or your other favorite vendor of choice.

Thanks
